ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CommandTracker.scala

```diff
@@ -45,7 +46,7 @@ import scala.util.{Failure, Success, Try}
   *
   */
 // TODO(mthvedt): This should have unit tests.
```

Comment:
Is this still relevant? Every class should have unit tests in general so I don't find this comment particularly useful.